test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -264,7 +275,19 @@ class test_ucp_worker_discard : public ucp_test {
         disconnect(sender());
     }
 
-    static void ep_destroy_func(uct_ep_h ep) {
+    static void ep_destroy_func(uct_ep_h ep)
+    {
+        for (std::vector<uct_completion_t*>::iterator iter = m_flush_comps.begin();
+             iter != m_flush_comps.end(); ++iter) {
+            ucp_request_t *req = ucs_container_of(*iter, ucp_request_t,
+                                                  send.state.uct_comp);
+            if (req->send.discard_uct_ep.uct_ep == ep) {
+                uct_invoke_completion(&req->send.state.uct_comp, UCS_ERR_CANCELED);
```

Comment:
can you please add comment "When UCT endpoint is destroyed, all its outstanding operations are completed with status UCS_ERR_CANCELED"

Reply:
done